vendor/bat-native-ledger/include/bat/ledger/public/interfaces/ledger.mojom

```diff
@@ -145,7 +145,6 @@ struct ExternalWallet {
   string type;
   string token;
   string address;
-  string linking_info;
```

Comment:
Drive by: this field is not needed by the bitFlyer work.

Reply:
Ah yeah, I think I reflexively added that to the wallet, good catch.